test/functional/forall/tests/test-forall-rangesegment.hpp

```diff
@@ -59,20 +59,20 @@ template <typename INDEX_TYPE, typename WORKING_RES, typename EXEC_POLICY,
   typename std::enable_if<std::is_signed<INDEX_TYPE>::value>::type* = nullptr>
 void runNegativeTests()
 {
-  ForallRangeSegmentFunctionalTest<INDEX_TYPE, WORKING_RES, EXEC_POLICY>(-5, 0);
-  ForallRangeSegmentFunctionalTest<INDEX_TYPE, WORKING_RES, EXEC_POLICY>(-5, 5);
+  ForallRangeSegmentTest<INDEX_TYPE, WORKING_RES, EXEC_POLICY>(-5, 0);
+  ForallRangeSegmentTest<INDEX_TYPE, WORKING_RES, EXEC_POLICY>(-5, 5);
 }
 
 
-TYPED_TEST_P(ForallFunctionalSegmentTest, RangeSegmentForall)
+TYPED_TEST_P(ForallSegmentTest, RangeSegmentForall)
```

Comment:
Do we want to make functional tests have the same name format of <Test>FunctionalTest to reflect the same naming standard of the unit tests?

Reply:
I thought about that. To me adding 'Functional' makes things too cumbersome. If it's not a unit test, then it's a functional test in my mind. But, I can change it back if you think it's important to keep it.

Reply:
No I think that's reasonable, just wanted to check